src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -11,6 +11,10 @@
     <PackageReference Include="jnm2.ReferenceAssemblies.net35" Version="1.0.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net35' and '$(TargetFramework)' != 'net40'">
```

Comment:
`IStructuralEquatable` is not available to NET4 on NuGet via `System.Runtime`. Instead, it is just a part of the framework